Sources/MapboxNavigation/InstructionsCardContainerView.swift

```diff
@@ -195,7 +195,9 @@ public class InstructionsCardContainerView: StylableView, InstructionsCardContai
         
         instructionsCardView.backgroundColor = backgroundColor
         instructionsCardView.separatorView.backgroundColor = separatorColor
-        instructionsCardView.primaryLabel.showHighlightedTextColor = shouldUseHighlightedColors
+        if instructionsCardView.primaryLabel.styleType == .day {
```

Comment:
We shouldn't check the `styleType`. Instead of the `style` checking, this PR has already added `isHighlighted` property to `GenericRouteShield` and `ExitView`. We should add the `highlightColor` to their appearance, and the `DayStyle` and `NightStyle`. So if the `isHighlighted` is `true`, they should apply the `highlightColor` to the `foregroundColor` and `borderColor`, otherwise. It's not something of the `Style`, it's to provide the color when the `InstructionLabel` is highlighted to the `GenericRouteShield` and `ExitView`.

Reply:
Also forgot to remove this when I included your suggested changes. Will fix!